.github/workflows/test.yml

```diff
@@ -243,4 +243,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: -- -D warnings
+          args: --tests -- -D warnings
```

Comment:
Shall we just do `--all-targets --all-features`?

Reply:
Hmmm, (to my surprise) it is actually different, as we have some tests that are behind features and those don't get processed with `--tests` only. There is one test that doesn't even compile if you use `--all-features`.